firmware/common/portapack_persistent_memory.hpp

```diff
@@ -192,9 +192,6 @@ void set_modem_repeat(const uint32_t new_value);
 uint32_t playing_dead();
```

Comment:
do we have to keep these two ?

Reply:
uint32_t playing_dead(); and void set_playing_dead(const uint32_t new_value); ?

Reply:
forgot it, thx!